deployment_script.py

```diff
@@ -10,12 +10,13 @@
 
 
 def check_version(mongoDB):
-    collection = mongoDB[VERSION_COLLECTION]
-    version_obj = collection.find_one()
-    if version_obj is None:
-        return True 
-    # return true if it is a newer db version
-    return db_version > version_obj['version']
+    return True
+    # collection = mongoDB[VERSION_COLLECTION]
+    # version_obj = collection.find_one()
+    # if version_obj is None:
+    #     return True 
+    # # return true if it is a newer db version
+    # return db_version > version_obj['version']
```

Comment:
I assume we don't want to merge this change?

Reply:
ugh...how many times do I have to make that mistake before I remember!